src/model/collection/util.ts

```
@@ -1,8 +1,12 @@
 /**
- * @param folder - the folder to sanitize.
- * @returns sanitized value with trailing slashes removed, if applicable.
+ * Returns a sanitized folder path by removing a trailing slash unless the path is the root ("/").
+ *
+ * The function trims whitespace from the input folder string. If the trimmed string equals "/", it is returned unchanged. Otherwise, any trailing slash is removed.
```

ESLint

[error] Expected only 0 line after block description

(jsdoc/tag-lines)